lib/moon/design/avatar.ex

```diff
@@ -27,56 +26,38 @@ defmodule Moon.Design.Avatar do
     ~F"""
     <div
       {=@id}
-      class={merge([
-        [
-          "overflow-hidden uppercase font-medium flex items-center justify-center bg-cover relative",
-          @bg_color,
-          @color,
-          set_border_radius(@size, @is_rounded),
-          "text-moon-10-caption h-6 w-6": @size == "xs",
-          "text-moon-12 h-8 w-8": @size == "sm",
-          "text-moon-14 h-10 w-10": @size == "md",
-          "text-moon-16 h-12 w-12": @size == "lg",
-          "text-moon-16 h-14 w-14": @size == "xl",
-          "text-moon-20 h-16 w-16": @size == "2xl"
-        ],
-        @class
-      ])}
+      class={[
+        merge([
+          [
+            "overflow-hidden uppercase font-medium flex items-center justify-center relative text-bulma bg-gohan",
+            set_border_radius(@size),
+            "text-moon-10-caption h-6 w-6": @size == "xs",
+            "text-moon-12 h-8 w-8": @size == "sm",
+            "text-moon-14 h-10 w-10": @size == "md",
+            "text-moon-16 h-12 w-12": @size == "lg",
+            "text-moon-16 h-14 w-14": @size == "xl",
+            "text-moon-20 h-16 w-16": @size == "2xl"
+          ],
+          @class
+        ]),
+        "bg-cover"
+      ]}
       style={style(assigns)}
       data-testid={@testid}
     >
-      <span :if={@name && !@image_url}>{@name}</span>
-      <Icon name="generic_user" class={set_icon_size(@size)} :if={!@name && !@image_url} />
-      <div
-        :if={@status_origin && @is_status_active}
-        class={
-          "absolute border-solid color-beerus rounded-full bg-roshi",
-          "top-0": @status_origin.vertical == "top",
-          "bottom-0": @status_origin.vertical == "bottom",
-          "left-0": @status_origin.horizontal == "left",
-          "right-0": @status_origin.horizontal == "right",
-          "w-2 h-2 border": @size == "xs",
-          "w-3 h-3 border-2": @size == "sm" || @size == "md",
-          "w-4 h-4 border-2": @size == "lg" || @size == "xl" || @size == "2xl"
+      <#slot />
+      <#slot
```

Comment:
please put back "name" property for backward compatibility. so, if no default slot given let's try to insert Icon component with corresponding slot here